src/task/script.js

```diff
@@ -303,3 +303,78 @@ function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
 }
 
 document.getElementById('visibity-hidden').classList.remove('visibity-hidden');
+
+const assigneeEl = document.getElementById('assignee');
+
+function debounce(func, delay) {
+  let searchDelay;
+  return (...args) => {
+    if (searchDelay) {
+      clearTimeout(searchDelay);
+    }
+
+    searchDelay = setTimeout(() => {
+      func(...args);
+    }, delay);
+  };
+}
+
+async function autoComplete(searchInput) {
+  try {
+    const response = await fetch(`${API_BASE_URL}/tasks`);
+    const data = await response.json();
+    removeAssignee();
+    if (searchInput !== '') {
+      const matches = data.tasks.filter((task) => {
+        if (task.assignee) {
+          return task.assignee
+            .toLowerCase()
+            .startsWith(searchInput.toLowerCase());
+        }
+      });
+      assigneeHTML(matches);
+    }
+  } catch {
+    alert('User not found');
+  }
+}
+
+function assigneeHTML(matches) {
```

Comment:
Whenever we use a search box say for a shopping site suggestions will pop up right so we want that kind of feature.
Please make changes to the following code accordingly